### PR TITLE
fix: fix builds on darwin for PHP <= 7.3

### PR DESCRIPTION
**Pull request by drupol**
Context: https://github.com/fossar/nix-phps/pull/201#discussion_r1087243962
Related PR: https://github.com/fossar/nix-phps/pull/201

### CODE REVIEW

**Comment by drupol** [comment]
It's failing... I don't get it.

File: pkgs/phps.nix [hunk 1]
```diff
@@ -43,8 +43,6 @@ let
 
               "--enable-libxml"
               "--with-libxml-dir=${prev.libxml2.dev}"
-            ]
-            ++ prev.lib.optionals (prev.lib.versionAtLeast args.version "7.3") [
               "--with-pcre-regex=${prev.pcre2.dev}"
```
Comment (jtojnar) [inline pkgs/phps.nix:46]: > checking for PCRE headers location... configure: error: Could not find pcre.h in /nix/store/r2cgszbs5mpjbrj0k28i9qa7wi7lqg15-pcre2-10.42-dev

Might be that older versions expect `${prev.pcre2.dev}/include`.

Comment (drupol) [reply pkgs/phps.nix]: Same issue

Comment (jtojnar) [reply pkgs/phps.nix]: We will need to look at the `configure.ac` or `m4` files, then.

Comment (drupol) [reply pkgs/phps.nix]: Then I'll happily leave this job to someone else, it's way beyond my knowledge at this moment.

Comment (drupol) [reply pkgs/phps.nix]: FYI, this issue is now:

image

Comment (drupol) [reply pkgs/phps.nix]: Same issue:

```2023-01-26T15:34:06.3265322Z configure: error: Couldn't find pcre2-config```

I also noticed this, I don't know if it's related:

image
2.

Comment (HelloWorld017) [reply pkgs/phps.nix]: As the `--with-external-pcre` is not recognized, the `--with-pcre-regex` is also added. So the error message is not relevant.

I'll take a look at the error next morning.

Comment (HelloWorld017) [reply pkgs/phps.nix]: I have succeeded this PR in #204 with fixes for this issue. Please take a look at it

**Pull request closed without merging**